src/components/backup/RestoreSheetFirstStep.js

```diff
@@ -38,13 +39,16 @@ const TitleRow = styled(RowWithMargins)`
   width: ${deviceWidth - 60};
 `;
 
-const RainbowText = styled(GradientText).attrs({
-  angle: false,
-  colors: colors.gradients.rainbow,
-  end: { x: 0, y: 0.5 },
-  start: { x: 1, y: 0.5 },
-  steps: [0, 0.774321, 1],
-})``;
+const RainbowText =
+  android && IS_TESTING
```

Comment:
You shouldn't treat `IS_TESTING` as a boolean since its value it's a string because it's an ENV var.
So IS_TESTING = 'false' would be evaluated as a string.

Just do `IS_TESTING === 'true'` instead